### PR TITLE
1096-notification adjustments

### PR DESCRIPTION
**Pull request by JCown**
Adjusted notifications toasts according to comments in 1096.
Bonus: Fixed toasts interval clearing (on toast close)

### CODE REVIEW

File: core/components/Notification.vue [hunk 1]
```diff
@@ -16,27 +16,26 @@ export default {
...
+      data.timeout = setTimeout(() => {
         this.action('close', this.notifications.length - 1)
       }, data.timeToLive || 5000)
+      this.notifications.push(data)
     })
   },
   methods: {
     action (action, id) {
-      this.$bus.$emit('notification-after-' + action, id)
+      this.$bus.$emit('cart-after-' + action, id)
```
Comment (pkarw) [inline core/components/Notification.vue]: Should be: notification-after- please :)

File: core/components/Notification.vue [hunk 2]
```diff
@@ -16,27 +16,26 @@ export default {
   },
   created () {
     this.$bus.$on('notification', data => {
-      if (this.notifications.length > 0 && this.notifications[this.notifications.length - 1].message === data.message) {
```
Comment (pkarw) [inline core/components/Notification.vue]: Why is that removed? It was to block the stacked set of the same messages which sometimes happen for example when user clicks 10times add to cart

**Comment by pkarw** [comment]
Please also take note of the https://github.com/DivanteLtd/vue-storefront/pull/1168 changes - You need to apply the changes to the core/components/notification.js instead of Notification.vue; Additionaly You need to remove the corePage / coreComponent references

**Comment by pkarw** [comment]
@JCown any chances to complete the changes requested?:)

**Comment by JCown** [comment]
sure! sorry but was quite busy last days, I'll work on that very soon :)
sorry once again for the delay

**Comment by pkarw** [comment]
Closing this PR for now - after providing the required changes please do open a new PR
